src/Conversion/Conversion.php

```diff
@@ -12,6 +12,11 @@ class Conversion
     protected $name = '';
 
     /**
+     * @var int
+     */
+    protected $extractDuration = 0;
```

Comment:
Rename this variable to something that makes it clear it is only used when working with video. Something like `extractVideoFrameAtSecond` or something better.